### ISSUE TITLE
You are using Nextcloud 20 but this release of the passwords app requires at least Nextcloud 21.

### ISSUE THREAD

**Issue by Django-BOfH** [marius-wieschollek/passwords#330]
### System Information
**Server:**
```
 Nextcloud Version: 20.0.5.2
 PHP Version: 7.2
 Database and Version: mysql
 Server OS and Version: CentOS 7.9
 Passwords Version: 2021.1.0
```

**Client:**
```
 Browser and Version: Firefox 84.0.2 (64-Bit
 Client OS and Version: CentOS 7.9
```

### Steps to reproduce
After updating my Nextcloud instance last day, I can't use the password plugin any more. :-1: 
### Expected result
The plugin should work! 

### Actual result
You are using Nextcloud 20 but this release of the passwords app requires at least Nextcloud 21. See https://git.mdns.eu/nextcloud/passwords/wikis/Administrators/Migrations/Minimum-Requirement-Error for help. 

Are you kidding me? Nextcloud 21? Have you got a time-amachine or a Tardes? :/ Today wie habe Nextcloud 20.0.5 and not 21!

**Comment by marius-wieschollek** [comment]
Sorry. Yesterday, there was a short period when a broken update was in the appstore. I will publish a fixed version later today.

**Comment by aszlig** [comment]
The same also (still) happens when you update from Nextcloud 19 (app version 2020.12.2) to Nextcloud 20 (app version 2021.1.0). Here is the log from my little automated upgrade test suite: https://headcounter.org/hydra/log/605cwg7pj70d4al4rigbj4n1jwiq3hpq-vm-test-run-nextcloud-upgrade.drv

**Comment by marius-wieschollek** [comment]
I think the issue here is that the during the upgrade the version number for nextcloud is still reported as 19 instead of 20.
Probably best to remove the check again altogether as the version requirement in Nextcloud should now work again.

**Comment by marius-wieschollek** [comment]
This should now be fixed with 2021.1.2
